app/controllers/physical_chassis_controller.rb

```diff
@@ -0,0 +1,43 @@
+class PhysicalChassisController < ApplicationController
+  include Mixins::GenericListMixin
+  include Mixins::GenericShowMixin
+  include Mixins::GenericSessionMixin
```

Comment:
You are providing your own `set_session_data` and `session_data` (instead of `get_session_data` that we have (I hope) everywhere else. In that case wou don't need to include `Mixins::GenericSessionMixin`.

May I suggest that you actually use the code from the mixin and don't write your own `{set,}_session_data`?

Reply:
I agree. I am oing to use only the `Mixins::GenericSessionMixin`.